### ISSUE TITLE
Stylelint doesn't seem to pick up inline less comments?

### ISSUE THREAD

**Issue by erikhallaraaker** [stylelint/stylelint#3761]
EDIT: I'm an idiot, apparently I'm not as good at searching as I thought, or even scrolling for that matter..
This is a duplicate of #3754 

> Clearly describe the bug

Seems like stylelint isn't picking up inline less comments, this is for several rules, but I included one of the easiest to reproduce.

Stylelint ignores block comments like `/* comment */`, but I'm not quite sure how it interprets `// comment`

> Which rule, if any, is the bug related to?

Several, but to mention a few
- block-closing-brace-newline-before
- block-closing-brace-newline-after
- declaration-block-semicolon-newline-after
- declaration-block-semicolon-newline-before

> What CSS is needed to reproduce the bug?

Well, in this case it's Less, but this reproduces:

```less
footer {
  display: block;
  font-size: 80%; // back to default font-size
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "declaration-block-semicolon-newline-after": "always-multi-line"
  }
}
```

> Which version of stylelint are you using?

`9.7.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint src/less/**/*.less`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to Less syntax

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The comment was 

```shell
typography.less
[stylelint] Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after) [declaration-block-semicolon-newline-after] (157, 24)
```

**Comment by jeddy3** [comment]
> I'm an idiot, apparently I'm not as good at searching as I thought, or even scrolling for that matter..

No worries. I mistitled the other issue (#3754) because I thought the regression was specific to the `declaration-block-semicolon-newline-after` rule, whereas it's more general than that.

I'll close this issue in favour of that one, though, so that we keep an discussion in one place.
